torchtext/csrc/vectors.cpp

```diff
@@ -11,36 +11,36 @@
 #include <sstream>
 #include <stdexcept>
 #include <string>
-#include <torch/script.h>
 #include <vectors.h>
 
 namespace torchtext {
 
-Vectors::Vectors(const IndexMap &stoindex, const torch::Tensor vectors,
+Vectors::Vectors(const IndexMap &stoi, const torch::Tensor vectors,
```

Comment:
So those changes are out of the pybind scope for this PR, right?

Reply:
You are right. I just created another PR with these changes https://github.com/pytorch/text/pull/921

Reply:
I think the crash is very likely coming from the modifications of this file.

Reply:
@Nayef211 did we merge those changes in a separate PR and the PR passes the CI test?

Reply:
Yes we did. I just merged in the latest master now

Reply:
> I think the crash is very likely coming from the modifications of this file.

Thanks @peterjc123, could you be more specific?